app/src/main/scala/org/alephium/explorer/service/BlockFlowClient.scala

```diff
@@ -147,9 +161,12 @@ object BlockFlowClient extends StrictLogging {
 
     def fetchBlocks(fromTs: TimeStamp,
                     toTs: TimeStamp,
-                    uri: Uri): Future[ArraySeq[ArraySeq[BlockEntity]]] = {
-      _send(getBlocks, uri, api.model.TimeInterval(fromTs, toTs))
-        .map(_.blocks.map(_.map(blockProtocolToEntity).toArraySeq).toArraySeq)
+                    uri: Uri): Future[ArraySeq[ArraySeq[BlockEntityWithEvents]]] = {
```

Comment:
should we deprecate `fetchBlocks` now?

Reply:
yes we could, nothing urgent as it's for internal use only.

Reply:
it's actually still used in our `SanityChecker`